algorithm/algorithm.go

```diff
@@ -0,0 +1,33 @@
+package algorithm
+
+import (
+	cpaconfig "github.com/jthomperoo/custom-pod-autoscaler/config"
+	"github.com/jthomperoo/custom-pod-autoscaler/execute"
+)
+
+const (
+	entrypoint   = "python"
+	shellTimeout = 10000
```

Comment:
imo a hard 10 second timeout is a little too restrictive. Some models, like ARIMA, will probably take longer to run. Also the more `storedValues` you have, the longer it'll take the model to run. What about 30 seconds?

Although, I'm not sure if the CPA would respect the longer timeout (haven't really looked at it)

Reply:
Yep good point, I've made a bunch of changes so that this value is configurable, and upped the default timeout to 30s.